website/docs/os.getSystemTags.md

```diff
@@ -0,0 +1,34 @@
+Returns the list of tags corresponding to a system.
+
+There are tags specific to each operating system (see [system()](system.md) for a complete list of identifiers.),
+and meta tags like `posix`, `darwin`, `desktop` and `mobile` tags.
+
+### Tags ###
+
+| OS       | Tags                                    |
+|----------|-----------------------------------------|
+| aix      | aix, posix, desktop                     |
```

Comment:
Android is missing from this list, probably because it's kept in a separate module.
https://github.com/premake/premake-core/blob/a5a00d656fe501a61744fde7eb0ce4c3b91acfb5/modules/android/_preload.lua#L43